FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheker/HealthCheckBackgroundService.cs

```diff
@@ -0,0 +1,65 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Fhir.Synapse.Common.Configurations;
+using Microsoft.Health.Fhir.Synapse.HealthCheker.Models;
+
+namespace Microsoft.Health.Fhir.Synapse.HealthCheker
+{
+    public class HealthCheckBackgroundService : BackgroundService
+    {
+        private static IHealthCheckEngine _healthCheckEngine;
+        private readonly ILogger<HealthCheckBackgroundService> _logger;
+        private TimeSpan _checkIntervalInSeconds = TimeSpan.FromSeconds(30);
```

Comment:
this value is assigned using the configuration value in constructor, don't need to initialize it here?